packages/sit-onyx/src/components/OnyxCalendar/OnyxCalendar.ct.tsx

```diff
@@ -117,3 +117,95 @@ test.describe("Screenshot tests (custom content)", () => {
     },
   });
 });
+
+test("range mode", async ({ mount, page }) => {
+  const minDate = getMockDate(-12);
+
+  // ARRANGE
+  const component = await mount(
+    <OnyxCalendar
+      selectionMode="range"
+      style={{ width: "40rem" }}
+      min={minDate}
+      showCalendarWeeks
+    />,
+  );
+  const startRange = component.getByRole("button", { name: "Tuesday, October 15," });
+  const endRange = component.getByRole("button", { name: "Monday, October 21," });
+  const endRange2 = component.getByRole("button", { name: "Friday, October 25," });
+  const middleDate = component.getByRole("button", { name: "Saturday, October 19," });
+  const today = component.getByRole("button", { name: "Wednesday, October 23," });
+  const weekNumber = component.getByRole("button", { name: "43" });
+  const weekNumberDisabled = component.getByRole("button", { name: "40" });
+
+  const startCell = component.getByRole("gridcell", { name: "Tuesday, October 15," });
+  const endCell2 = component.getByRole("gridcell", { name: "Friday, October 25," });
+  const todayCell = component.getByRole("gridcell", { name: "Wednesday, October 23," });
+
+  // Today outside range
+  // ACT
+  await startRange.click();
+  await endRange.click();
+  await page.mouse.move(0, 0);
+  //ASSERT
+  await expect(component).toHaveScreenshot("range.png");
+  // Today inside range
+  // ACT
+  await startRange.click();
+  await endRange2.click();
+  await page.mouse.move(0, 0);
+  // ASSERT
+  await expect(startCell).toHaveClass(/onyx-calendar-cell--range-start/);
+  await expect(endCell2).toHaveClass(/onyx-calendar-cell--range-end/);
+  await expect(todayCell).toHaveClass(/onyx-calendar-cell--range-middle/);
```

Comment:
```suggestion
  await expect(startCell).toContainClass("onyx-calendar-cell--range-start");
  await expect(endCell2).toContainClass("onyx-calendar-cell--range-end");
  await expect(todayCell).toContainClass("onyx-calendar-cell--range-middle");
```

We can use `.toContainClass()` so we don't need a regex